tmol/score/ref/ref_energy_term.py

```diff
@@ -0,0 +1,67 @@
+import torch
+import numpy
+
+from ..energy_term import EnergyTerm
+
+from tmol.database import ParameterDatabase
+from tmol.score.ref.ref_whole_pose_module import RefWholePoseScoringModule
+
+from tmol.chemical.restypes import RefinedResidueType
+from tmol.pose.packed_block_types import PackedBlockTypes
+from tmol.pose.pose_stack import PoseStack
+
+from collections import OrderedDict
+
+
+class RefEnergyTerm(EnergyTerm):
+    device: torch.device  # = attr.ib()
+
+    def __init__(self, param_db: ParameterDatabase, device: torch.device):
+        super(RefEnergyTerm, self).__init__(param_db=param_db, device=device)
+
+        self.ref_weights = param_db.scoring.ref.weights
+        self.device = device
+
+    @classmethod
+    def score_types(cls):
+        import tmol.score.terms.ref_creator
+
+        return tmol.score.terms.ref_creator.RefTermCreator.score_types()
+
+    def n_bodies(self):
+        return 1
+
+    def setup_block_type(self, block_type: RefinedResidueType):
+        super(RefEnergyTerm, self).setup_block_type(block_type)
+
+        print(block_type.name3)
+        ref_weight = getattr(self.ref_weights, block_type.name3)
```

Comment:
Is name3 appropriate here?

Reply:
Ok, I think github may be acting up for me or something. I was looking at this PR yesterday and did not see any comments from Frank, but now I see this comment from 2 days ago (along with some others).

I'll go ahead and make the change for adding the extra entries for HIS_D/CYD and changing it to base_name.